src/controllers/itemDetails/index.js

```diff
@@ -1718,15 +1718,22 @@ function renderCollectionItemType(page, parentItem, type, items) {
     imageLoader.lazyChildren(collectionItems.lastChild);
 }
 
-function renderMusicVideos(page, item, user) {
-    ServerConnections.getApiClient(item.ServerId).getItems(user.Id, {
+function renderMusicVideos(page, item, user, isAlbum = false) {
+    const request = {
         SortBy: 'SortName',
         SortOrder: 'Ascending',
         IncludeItemTypes: 'MusicVideo',
         Recursive: true,
-        Fields: 'PrimaryImageAspectRatio,BasicSyncInfo,CanDelete,MediaSourceCount',
-        AlbumIds: item.Id
-    }).then(function (result) {
+        Fields: 'PrimaryImageAspectRatio,BasicSyncInfo,CanDelete,MediaSourceCount'
+    };
+
+    if (isAlbum) {
```

Comment:
We can check `item.Type == 'MusicAlbum'` here instead of passing it through the argument. Bill, what do you think?